frontend/ui/detailview.jsx

```diff
@@ -155,12 +166,12 @@ class DetailViewComponent extends React.Component {
           seriesMap[build.version] = {};
         }
         build.data.forEach((datum) => {
-          const date = datum[0];
+          const date = this.state.relative ? datum[0] / 60.0 / 60.0 : new Date(datum[0]);
```

Comment:
In the case that you pass ?relative=1 to the API endpoint (see the python code), we return the timestamp as an integer indicating seconds passed since the release as opposed to a date.